containers/Uniswap.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from "react";
+
+import { useQuery } from "@apollo/client";
+import { PAIR } from "../apollo/uniswap/queries";
+
+const useUniswap = ({ token0, token1 }: { token0: string; token1: string }) => {
```

Comment:
Is this used anywhere?

Reply:
not used yet, but it will be to get our "fair value" for the perp